org/org.yaml

```diff
@@ -724,3 +724,19 @@ orgs:
         members:
         - afrittoli
         privacy: closed
+      resolution.maintainers:
+        description: The resolution maintainers
+        maintainers:
+        - sbwsg
+        - dibyom
```

Comment:
`maintainers` is the maintainers of the GitHub team. I think using `members` would be the appropriate field to use.
Unfortunately peribolos (or GitHub?) does not allow for an empty `maintainers` list - which would be ideal because we want the teams to be maintained through periobolos only. For other teams I used members of the governance team as maintainers of teams, but that too is not ideal, because members of the GB change over time.
Perhaps we could use a bot account as `maintainers`?

Reply:
I've added the governing board members as maintainers on the teams for now until a bot account is set up.